src/librustc_passes/hir_stats.rs

```diff
@@ -51,8 +51,8 @@ pub fn print_hir_stats(krate: &hir::Crate) {
 pub fn print_ast_stats<'v>(krate: &'v ast::Crate, title: &str) {
     let mut collector = StatCollector {
```

Comment:
`StatCollector::default()`

Reply:
Not worth it, I'd rather keep it symmetrical with `print_hir_stats`.